src/vmm/src/cpu_config/x86_64/static_cpu_templates/t2s.rs

```diff
@@ -119,20 +212,52 @@ pub fn t2s() -> CustomCpuTemplate {
                 leaf: 0x80000008,
                 subleaf: 0x0,
                 flags: KvmCpuidFlags(0),
-                modifiers: vec![CpuidRegisterModifier {
-                    register: CpuidRegister::Ebx,
-                    bitmap: RegisterValueFilter {
-                        filter: 0b00000000000000000000001000000000,
-                        value: 0b00000000000000000000000000000000,
+                modifiers: vec![
+                    // EBX:
+                    // - Bit 09: WBNOINVD
+                    CpuidRegisterModifier {
+                        register: CpuidRegister::Ebx,
+                        bitmap: RegisterValueFilter {
+                            filter: 0b0000_0000_0000_0000_0000_0010_0000_0000,
+                            value: 0b0000_0000_0000_0000_0000_0000_0000_0000,
+                        },
                     },
-                }],
+                ],
             },
         ],
-        msr_modifiers: vec![RegisterModifier {
+        msr_modifiers: vec![
+            // IA32_ARCH_CAPABILITIES:
+            // - Bit 00: RDCL_NO
+            // - Bit 01: IBRS_ALL
+            // - Bit 02: RSBA
+            // - Bit 03: SKIP_L1DFL_VMENTRY
+            // - Bit 04: SSB_NO
+            // - Bit 05: MDS_NO
+            // - Bit 06: IF_PSCHANGE_MC_NO
+            // - Bit 07: TSX_CTRL
+            // - Bit 08: TAA_NO
+            // - Bit 09: Reserved
+            // - Bit 10: MISC_PACKAGE_CTRLS
```

Comment:
* I am ok with leaving `IA32_MISC_PACKAGE_CTRLS` as is. There isn't a clear consensus on this with a look over the existing docs. The bit is used for [running average power limits (RAPL)](https://www.intel.com/content/www/us/en/developer/articles/technical/software-security-guidance/advisory-guidance/running-average-power-limit-energy-reporting.html). I don't know `MISC_PACKAGE_CTLS` was Intel's original intent or not, but it seems there was a change in the Linux codebase to `_CTRLS` to be more consistent with other registers naming convention. This may have something to do with the conflict in naming conventions.
* Bit 9 has been updated to enumerate memory controls, see item [23. Updates to Chapter 2, Volume 4](https://cdrdv2.intel.com/v1/dl/getContent/671294). Bit 9 now enumerates support for `MCU_CONTROL`.

Reply:
good catch. thanks. will update.